randovania/games/dread/json_data/Cataris.txt

```diff
@@ -2974,6 +2977,8 @@ Extra - asset_id: collision_camera_036
   * Extra - actor_def: actordef:actors/props/breakablemag007/charclasses/breakablemag007.bmsad
   > Upper-right Corner
       Trivial
+  > Door to Central Unit
+      Shoot Beam
```

Comment:
Events only go back trivially to a single node. They are handled a bit differently. This connection should be removed.